llvm/lib/CodeGen/LiveDebugValues/InstrRefBasedImpl.h

```diff
@@ -1003,38 +1041,44 @@ using OverlapMap =
 /// identified.
 class VLocTracker {
 public:
+  /// Ref to function-wide map of DebugVariable <=> ID-numbers.
+  DebugVariableMap &DVMap;
   /// Map DebugVariable to the latest Value it's defined to have.
   /// Needs to be a MapVector because we determine order-in-the-input-MIR from
-  /// the order in this container.
+  /// the order in this container. (FIXME: this is less true now).
```

Comment:
Imo it could be helpful to future developers if you explained why it's less true now here?